conf/script/build-ha-io

```diff
@@ -260,19 +260,19 @@ net_config_check() {
 # rvolume is mounted at /var/motr2.
 prepare_var_motr_dirs() {
     log "${FUNCNAME[0]}: make sure there are no stale mounts hanging..."
-    mountpoint -q /var/motr1 && sudo umount /var/motr1 || true
+    ssh $lnode 'mountpoint -q /var/motr1 && sudo umount /var/motr1 || true'
```

Comment:
why not use run_on_both function for all of such commands?

Reply:
Because the commands are different?